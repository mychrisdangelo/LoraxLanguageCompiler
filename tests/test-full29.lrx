int main()
{
	tree <int>t(2);

	t = 1[2, 3[4, 5]];
	t%0%1 = 102[103, 104];

	print("t = ", t, "\n");

}