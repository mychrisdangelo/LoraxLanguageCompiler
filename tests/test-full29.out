t = 1[2[null,102[103[null,null],104[null,null]]],3[4[null,null],5[null,null]]]
